app/src/main/res/layout/submitted_answer_item.xml

```diff
@@ -67,26 +67,35 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
 
-    <androidx.recyclerview.widget.RecyclerView
-      android:id="@+id/submitted_answer_recycler_view"
+    <LinearLayout
+      android:id="@+id/submitted_answer_recycler_view_container"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:background="@drawable/submitted_answer_background"
-      android:paddingStart="8dp"
-      android:paddingEnd="8dp"
-      android:visibility="gone"
-      app:itemDecorator="@{@drawable/divider}"
-      app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+      android:orientation="vertical"
+      android:contentDescription="@{viewModel.isCorrectAnswer ? @string/correct_submitted_answer : @string/incorrect_submitted_answer}"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"
-      app:list="@{submittedListAnswer.setOfHtmlStringsList}" />
+      app:layout_constraintTop_toTopOf="parent">
+
+      <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/submitted_answer_recycler_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/submitted_answer_background"
+        android:paddingStart="8dp"
+        android:paddingEnd="8dp"
+        android:visibility="gone"
+        app:itemDecorator="@{@drawable/divider}"
+        app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+        app:list="@{submittedListAnswer.setOfHtmlStringsList}" />
+    </LinearLayout>
 
     <ImageView
       android:id="@+id/answer_tick"
       android:layout_width="@dimen/answer_tick_size"
       android:layout_height="@dimen/answer_tick_size"
       android:layout_gravity="center_vertical"
       android:layout_marginStart="@dimen/answer_tick_margin"
+      android:importantForAccessibility="no"
```

Comment:
This icon description is not important because we are already appending `Correct` when needed.

Reply:
okay